lorien/InfiniteCanvas/InfiniteCanvas.gd

```diff
@@ -345,4 +346,13 @@ func _undo_delete_stroke(stroke: BrushStroke) -> void:
 
 # -------------------------------------------------------------------------------------------------
 func _on_window_resized() -> void:
-	_viewport.size = get_viewport_rect().size
+	# Multiplying by scale needed to fix canvas after _set_scale() workaround
+	_viewport.size = get_viewport_rect().size * _scale
+
+# -------------------------------------------------------------------------------------------------
+func _set_scale(scale: float) -> void:
```

Comment:
Only private methods should start with an underscore to distinguish them from public ones. `set_scale` is used in `Main.gd`, so it's public.